test/Cimpress.Extensions.Http.Caching.InMemory.UnitTests/InMemoryCacheFallbackHandler_when_sending_request.cs

```diff
@@ -42,12 +42,29 @@ public class InMemoryCacheFallbackHandler_when_sending_request
 
             // execute twice, validate cache is called each time
             await client.GetAsync(url);
-            cache.Verify(c => c.CreateEntry(url), Times.Once);
+            cache.Verify(c => c.CreateEntry(HttpMethod.Get + url), Times.Once);
             await client.GetAsync(url);
-            cache.Verify(c => c.CreateEntry(url), Times.Exactly(2));
+            cache.Verify(c => c.CreateEntry(HttpMethod.Get + url), Times.Exactly(2));
         }
 
         [Fact]
+        public async Task Updates_the_cache_for_head_and_get_independently()
+        {
+            // setup
+            var testMessageHandler = new TestMessageHandler();
+            var cache = new Mock<IMemoryCache>(MockBehavior.Strict);
+            var cacheTime = TimeSpan.FromSeconds(123);
+            cache.Setup(c => c.CreateEntry(url));
```

Comment:
No, but I wanted to adjust the headline of the test. Yes, I could go through a couple more send options like `PATCH` or `OPTIONS` but I opted to skip that.